assets/evm/v1/assets_dev.json

```diff
@@ -11,5 +11,445 @@
+            }
+        ]
+    },
+    {
+        "symbol": "CONV",
+        "precision": 18,
+        "name": "Convergence",
+        "priceId": "convergence",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/CONV.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0x8006320739fC281da67Ee62eB9b4Ef8ADD5C903a"
+            }
+        ]
+    },
+    {
+        "symbol": "GLINT",
+        "precision": 18,
+        "name": "Beamswap Token",
+        "priceId": "beamswap",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/Default.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0xcd3B51D98478D53F4515A306bE565c6EebeF1D58"
+            }
+        ]
+    },
+    {
+        "symbol": "EFT",
+        "precision": 18,
+        "name": "Energyfi Token",
+        "priceId": "energyfi",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/EFT.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0xA423E7eEB60547d9C7b65005477b63ae7CE67E62"
+            }
+        ]
+    },
+    {
+        "symbol": "WETH",
+        "precision": 18,
+        "name": "Wrapped Ether",
+        "priceId": "wrapped-ethw",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/ETH.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0xab3f0245B83feB11d15AAffeFD7AD465a59817eD"
+            }
+        ]
+    },
+    {
+        "symbol": "USDT",
+        "precision": 6,
+        "name": "Tether USD",
+        "priceId": "tether",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/USDT.svg",
+        "instances": [
+            {
+                "chainId": "401a1f9dca3da46f5c4091016c8a2f26dcea05865116b286f60f668207d1474b",
+                "contractAddress": "0xB44a9B6905aF7c801311e8F4E76932ee959c663C"
+            }
+        ]
+    },
+    {
+        "symbol": "USDC",
+        "precision": 6,
+        "name": "USD Coin",
+        "priceId": "usd-coin",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/USDC.svg",
+        "instances": [
+            {
+                "chainId": "401a1f9dca3da46f5c4091016c8a2f26dcea05865116b286f60f668207d1474b",
+                "contractAddress": "0xE3F5a90F9cb311505cd691a46596599aA1A0AD7D"
+            }
+        ]
+    },
```

Comment:
We should combine the same assets into a single element with multiple "instances"
For example, instances for USDC should contain two elements - one for Moonbeam and another for Moonriver. We should also apply same logic to other overlapping tokens

Reply:
combined instances of same token